Frontend/ui-driver/src/Screens/DriverProfileScreen/View.purs

```diff
@@ -1100,11 +1102,46 @@ getRcDetails state = do
      else [])
   <>[{ key :(getString MODEL_NAME) , value : if config.rcStatus then config.rcDetails.vehicleModel else Just "NA", action :  NoAction , isEditable : false}
   , { key : (getString COLOUR) , value : if config.rcStatus then config.rcDetails.vehicleColor else Just "NA", action :NoAction , isEditable : false } ]
-  <> (if not null state.data.rcDataArray then
-      [{ key : "" , value : Just (getString EDIT_RC), action :UpdateRC config.rcDetails.certificateNumber config.rcStatus , isEditable : false }]
-    else [])
+  -- <> (if not null state.data.rcDataArray then
+  --     [{ key : "" , value : Just (getString EDIT_RC), action :UpdateRC config.rcDetails.certificateNumber config.rcStatus , isEditable : false }]
+  --   else [])
   )
 
+bottomPill :: forall w. ST.DriverProfileScreenState -> (Action -> Effect Unit) -> PrestoDOM (Effect Unit) w
+bottomPill state push = 
+  linearLayout[
+    width MATCH_PARENT,
+    height WRAP_CONTENT,
+    padding $ PaddingVertical 20 20
+  ][
+    linearLayout [
+      width $ V (((screenWidth unit)-32)/2) ,
```

Comment:
try to minimize the use of brackets

Reply:
sure but here all 3 brackets are needed. I just replaced with $ now

Reply:
minimize use of brackets includes using $ only